project/templates/homePage.php

```diff
@@ -1,3 +1,18 @@
+<?php
+include_once "../utils/sessionUtils.php";
+include_once "../database/userUtils.php";
+if(getActiveUserID__FAKE()==null){
+    header("Location: login.php");
```

Comment:
use loginRequired() from sessionUtils.php instead as it displays a message why the user was redirected